ui/server/src/main/scala/pl/touk/nussknacker/ui/db/entity/ProcessActionEntityFactory.scala

```diff
@@ -0,0 +1,75 @@
+package pl.touk.nussknacker.ui.db.entity
+
+import java.sql.Timestamp
+import java.time.LocalDateTime
+
+import pl.touk.nussknacker.restmodel.processdetails.ActionType
+import pl.touk.nussknacker.restmodel.processdetails.ActionType.ActionType
+import pl.touk.nussknacker.ui.util.DateUtils
+import slick.ast.BaseTypedType
+import slick.jdbc.{JdbcProfile, JdbcType}
+import slick.lifted.{ForeignKeyQuery, ProvenShape, TableQuery => LTableQuery}
+import slick.sql.SqlProfile.ColumnOption.{NotNull, Nullable}
+
+trait ProcessActionEntityFactory {
+
+  protected val profile: JdbcProfile
+  import profile.api._
+
+  implicit def deploymentMapper: JdbcType[ActionType] with BaseTypedType[ActionType] =
+    MappedColumnType.base[ActionType, String](_.toString, ActionType.withName)
+
+  val processActionsTable: LTableQuery[ProcessActionEntityFactory#ProcessActionEntity] =
+    LTableQuery(new ProcessActionEntity(_))
+
+  val processVersionsTable: LTableQuery[ProcessVersionEntityFactory#ProcessVersionEntity]
+  val commentsTable: LTableQuery[CommentEntityFactory#CommentEntity]
+  val environmentsTable: LTableQuery[EnvironmentsEntityFactory#EnvironmentsEntity]
```

Comment:
is it currently used? maybe TODO for remove?

Reply:
Removed.